config.yml

```diff
@@ -1,8 +1,12 @@
 azure_iot:
-  hostname: ""
-  device_id: ""
-  shared_access_key: ""
+  hostname: "iothub-oor-ont-weu-itr-01.azure-devices.net"
+  device_id: "test-diana"
+  shared_access_key: "{SHARED_ACCESS_KEY_IOT}"
```

Comment:
let's rename this to device_connection_string

Reply:
It's actually only the shared access key, the connection string is created dinamically.